docs/reference/configuration.md

```diff
@@ -238,9 +238,21 @@ The following key formats are supported:
 - PKCS#8 PEM or DER-encoded RSA or ECDSA private key, encrypted or not
 - SEC1 PEM or DER-encoded ECDSA private key
 
+The signing keys are used for signing ID Tokens (as returned in the [Token Endpoint]
```

Comment:
They are also used if an upstream is configured to use the `private_key_jwt` client auth method; in which case, MAS uses one of those private keys to sign a JWT that the upstream OAuth provider can verify for auth. But it's a bit of a niche use case, not necessarily worth mentioning